src/Components/Aspire.NATS.Net/AspireNatsClientExtensions.cs

```diff
@@ -19,6 +19,7 @@ namespace Microsoft.Extensions.Hosting;
 public static class AspireNatsClientExtensions
 {
     private const string DefaultConfigSectionName = "Aspire:NATS:Net";
+    private const string ActivityNameSource = "NATS.Net";
```

Comment:
Can you update 

* https://github.com/dotnet/aspire/blob/main/src/Components/Telemetry.md
* https://github.com/dotnet/aspire/blob/main/src/Components/Aspire_Components_Progress.md

With this information?